src/scripts/components/icons/copy-icon.tsx

```diff
@@ -0,0 +1,11 @@
+import React, {FunctionComponent} from 'react';
+
+export const CopyIcon: FunctionComponent = () => (
+  <svg
+    width="23"
```

Comment:
All our icons should be 24x24 (if not 24px they should have at least quadratic dimensions)